src/components/pages/blog/posts-list/posts-list.jsx

```diff
@@ -23,16 +22,15 @@ const PostsList = ({ items }) => (
             </h1>
             <div className="mt-5 flex items-center justify-between 2xl:mt-4">
               <div className="flex items-center">
-                <img
+                <GatsbyImage
                   className="w-10 shrink-0 rounded-full"
-                  src={POST_AUTHORS[author]?.photo}
-                  alt={POST_AUTHORS[author]?.name}
+                  imgClassName="rounded-full"
+                  image={getImage(author.postAuthor.image.localFile)}
```

Comment:
I think it could be replaced with this
getImage(author.postAuthor.image) no need for passsing .localFile.

<img width="1015" alt="screenshot-2022-07-14-12-11-01@2x" src="https://user-images.githubusercontent.com/2697570/178959099-8c16e38b-a610-4ef7-b776-35807e4109d2.png">